deployment/terraform/cluster/aws/apps/variables.tf

```diff
@@ -262,6 +262,7 @@ variable "ad-manager" {
     aquila_uri = "http://aquila-detector/detect"
     models_region = "us-west-2"
     models_bucket = "aa-models"
+    modelservice_uri_template = "http://modelservice/api/detectors/search/findTopByDetectorUuidOrderByBuildTimestampDesc?uuid=%s"
```

Comment:
Whoa, that's some search endpoint... :D

Take a look at the @Query annotation.